lib/admin_plugins/cleanstatusdb.js

```diff
@@ -0,0 +1,69 @@
+'use strict';
+
+var cleanstatusdb = {
+  name: 'cleanstatusdb'
+  , label: 'Clean mongo status database'
+  , pluginType: 'admin'
+};
+
+function init() {
+  return cleanstatusdb;
+}
+
+module.exports = init;
+
+cleanstatusdb.actions = [
+    {
+        name: 'Delete all documents from devicestatus collection'
+      , description: 'This task removes all documents from devicestatus collection. Useful when uploader battery status is not properly updated'
+      , buttonLabel: 'Delete all documents'
+      , confirmText: 'Delete all documents from devicestatus collection?'
+    }
+  ];
+
+cleanstatusdb.actions[0].init = function init(client, callback) {
+  var translate = client.translate;
+  var $status = $('#admin_' + cleanstatusdb.name + '_0_status');
+  
+  $status.hide().text(translate('Loading database ...')).fadeIn('slow');
+  $.ajax('/api/v1/devicestatus.json?count=500000', {
```

Comment:
yeah. this plugin is an example not written much efficient way. I checked it on my database (never cleaned) and it was something like 5MB. no so bad